### ISSUE TITLE
[Bug] Broken math display when exporting html

### ISSUE THREAD

**Issue by anti-destiny** [Zettlr/Zettlr#3228]
### Description

Thank you for your great markdown editor. 

Recently, I found I can't export my notebook correctly with the latest Zettlr (version 2.2.1), downloaded from the official site. 

The math equations looks alright in Zettlr:
![Correctly rendered in Zettlr](https://user-images.githubusercontent.com/24856363/154389363-b5051f96-3e2c-439e-846f-1f4e2dbb317e.png "Correctly rendered in Zettlr")
However, when exported to HTML, the equations can not be correctly displayed: 
![Broken in Chrome](https://user-images.githubusercontent.com/24856363/154389428-9fd511ea-70da-45e8-9505-838527b2ae91.png "Broken in Chrome")

### Reproducing

Just install Zettlr 2.2.1, then export a markdown file with math equations with default setting. 

### Zettlr Version

Stable (most recent version)

### Specify version

2.2.1

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 10

### Additional Information

In addition, I found that if I replace the scripts in the exported HTML with the official mathjax files, I can fix this issue.

```html
<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
```
Hope this would be helpful.

**Comment by nathanlesage** [comment]
This could indicate that one needs to set `standalone: true` in the defaults file for HTML exports. Does that fix the issue?

**Comment by anti-destiny** [comment]
Setting `standalone: true` in the defaults file improves the situation. However, there still remain some pitfalls:
![image](https://user-images.githubusercontent.com/24856363/154600638-6c51be62-1810-4e87-9283-02f1becea1bb.png)

Now I exported my markdown manually using pandoc commands. And the result looks better:
![image](https://user-images.githubusercontent.com/24856363/154600912-50ecdf49-bd6a-4276-85e2-8bf1637d5e6d.png)



> This could indicate that one needs to set `standalone: true` in the defaults file for HTML exports. Does that fix the issue?

**Comment by nathanlesage** [comment]
Well, this now looks as if you are simply missing the corresponding fonts, are you?

**Comment by anti-destiny** [comment]
> Well, this now looks as if you are simply missing the corresponding fonts, are you?

Yes, I guess the problem resides in fonts. However, I think fonts are dynamically loaded by mathjax, so I don't need to maintain any fonts on my computer. Am I right?

**Comment by nathanlesage** [comment]
I actually don't know :S I only know that one does have to have special fonts for this. Are there some differences in the Markup being generated for both versions?

**Comment by nathanlesage** [comment]
Any updates on this?

**Comment by nathanlesage** [comment]
Closing due to lack of activity

**Comment by wakita** [comment]
I am new to Zettlr.  I experienced the same when I exported an HTML in a Temporary directory.

Is the URL starting with file:///output/ correct?

~~~
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_Main-Italic.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_Math-BoldItalic.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_Size1-Regular.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_Size2-Regular.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_Size3-Regular.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_Size4-Regular.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_AMS-Regular.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_Calligraphic-Regular.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_Calligraphic-Bold.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_Fraktur-Regular.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_Fraktur-Bold.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_SansSerif-Regular.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_SansSerif-Bold.woff
[Error] Not allowed to load local resource: file:///output/chtml/fonts/woff-v2/MathJax_SansSerif-Italic.woff
~~~

**Comment by wakita** [comment]
I think I resolved this issue by adjusting the default settings for exporting HTML files.  Here follows what I have done:

1. Start Zettlr
2. From the Zettlr menu, select "Assets Manager"
3. Choose "HTML"
4. Modify `url` attribute of the `html-math-method` from the default settings to `https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js`.

I found this URL on the following page: [Configuring and Loading MathJax](https://docs.mathjax.org/en/latest/web/configuration.html) from official MathJax documentation.

Hope this is helpful.
